server/circleci.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package server
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-mattermod/model"
+	"github.com/mattermost/mattermost-server/mlog"
+
+	"github.com/cpanato/go-circleci"
+)
+
+func (s *Server) triggerCircleCiIfNeeded(pr *model.PullRequest) {
+	client := &circleci.Client{Token: s.Config.CircleCIToken}
+
+	if strings.Contains(pr.FullName, "mattermost/") {
```

Comment:
`strings.Contains` seems like it might match things that aren't from the `mattermost` repo, e.g. `github.com/lieut-data/mattermost`?

Reply:
Agree, thanks for catching that.
the `Fullname` is something like this `lieu-data/reepo1` there is no `/` in the end. but I will update this to another way to check

Reply:
@lieut-data PTAL